### PR TITLE
08 post a new comment

### PR DESCRIPTION
**Pull request by tuttitaken01**
The user can now post a comment but it doesn't appear straight away (still figuring this one out)

### CODE REVIEW

**Review by itsaliabdallah — CHANGES_REQUESTED** [review]
Please implement optimisc rendering

File: src/components/AddCom.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            setList([...people]);
+        })
+    }, [avatar]);
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        setMsg('Comment posted successfully!');
+        setError(null);
+        setText('');
+        axios.post(`https:gameview.onrender.com/api/reviews/${reviewId}/comments`, {
```
Comment (MrMcSnail) [inline src/components/AddCom.jsx:28]: YOu might want to extract this to a separate api/utils module.

File: src/components/AddCom.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            username: avatar, body: text,
+        })
+        .then(({data}) => {
+            return data.comment[0];
+        })
+        .then((commentFromApi) => {
```
Comment (MrMcSnail) [inline src/components/AddCom.jsx:34]: This should be optimistically rendered an dthe setting of state not dependant on the post promise resolving.

**Comment by tuttitaken01** [comment]
Edited the addcomment file

**Review by itsaliabdallah — APPROVED** [review]
nice!

**Pull request merged**